### PR TITLE
Updated logic to get api_client of niaas setup

### PR DESCRIPTION
**Pull request by sbhagwatvnera**

### CODE REVIEW

File: examples/init_api_client.py [hunk 1]
```diff
@@ -26,9 +42,32 @@ def get_api_client(args):
...
+                        "?refresh_token={}".format(args.refresh_token)
+    response = requests.post(authorize_api_url)
+    response = json.loads(str(response.text))
+    csp_auth_token = response['access_token']
+    return csp_auth_token
+
 
 def parse_arguments():
     parser = argparse.ArgumentParser(description='Run Public APIs on vRNI Platform')
+    parser.add_argument("--deployment_type", action="store",
```
Comment (akulkarnivnera) [inline examples/init_api_client.py]: Lets have default as on-prem

File: examples/init_api_client.py [hunk 2]
```diff
@@ -37,13 +76,15 @@ def parse_arguments():
                         default='admin', help="password for authentication")
     parser.add_argument("--domain_type", action="store",
                         default='LOCAL', help="domain type for authentication")
-    parser.add_argument("--data_sources_csv", action="store",
-                        default='data_sources.csv', help="Name of CSV file")
-    args = parser.parse_args()
-    return args
+
+    # Network Insight as a service (NIAAS) parameters
+    parser.add_argument('--refresh_token', action='store',
```
Comment (akulkarnivnera) [inline examples/init_api_client.py]: We need in detail steps to get refresh_token for NIaaS setups

File: examples/init_api_client.py (hunk 1 above)
Comment (sbhagwatvnera) [reply examples/init_api_client.py]: Fixed.

File: examples/init_api_client.py (hunk 2 above)
Comment (sbhagwatvnera) [reply examples/init_api_client.py]: Updated.

**Pull request merged**